scripts/youtrack2csv.py

```diff
@@ -0,0 +1,9 @@
+import pandas as pd
```

Comment:
This might confuse people at some point, offering two methods while JQ can do the job might be redundant.

Reply:
That's true, but the python script is much more user friendly to reorganize. JQ is substantively harder to use. Either way, the script will be useless pretty quickly as IOHKS tickets are no longer containing the relevant log-classifier text data and will soon not exist.